### DISCUSSION TITLE
Customizing Chrome preferences

### DISCUSSION THREAD

**Discussion by alexnguyennz** [wxt-dev/wxt#964] [Q&A] [answered]
Hi I'm wanting to customize the Chrome instance e.g. open the dock on the bottom - how would I do this if possible with WXT? Thanks.

**Comment by aklinker1** [comment, marked as answer]
You can customize any setting in your profile's `Preferences` file ( `/path/to/profile/Preferences`) via: https://wxt.dev/api/reference/wxt/interfaces/ExtensionRunnerConfig.html#chromiumpref

> Docs may be out of date... Instead of using a nested object as shown in the examples (`{ download: { default_directory: ... } }`), you may have to use a flat key (`{ "download.default_directory": ... }`)

The other option is to pass CLI flags: https://wxt.dev/api/reference/wxt/interfaces/ExtensionRunnerConfig.html#chromiumargs

I don't know which of these options you should use for your case. Or maybe neither will work and there's not a way to configure what you're trying to do 🤷 chrome isn't very good at allowing customization on startup

**Comment by alexnguyennz** [comment]
Thanks. Both nested and flat work (although chromiumPref is expecting a string).

**Reply by aklinker1** [reply]
Woops, will get that fixed

**Reply by aklinker1** [reply]
Fixed in https://github.com/wxt-dev/wxt/commit/fda1e18f4820471af661239e20711102533f246b
